utils/plugin_docs.py

```diff
@@ -0,0 +1,56 @@
+
+import os
+from distutils.dir_util import copy_tree
+from jinja2 import Environment, FileSystemLoader
+
+
+DOCS_DIR = 'docs'
```

Comment:
I would put a comment here or rename the constant to make it more clear that this is a plugin documentation directory.